docs/src/prototypes/CopyToClipboard/index.tsx

```diff
@@ -27,27 +27,37 @@ const CopyToClipboardPrototype: React.FC<CopyToClipboardPrototypeProps> = props
 }
 
 const CopyToClipboardInMenu: React.FC = props => {
```

Comment:
Let's keep the original example, and add a new example demonstrating the `targetRef` prop.

Reply:
why? we already have the example for not attached, no need to duplicate